### ISSUE TITLE
QR code scanning doesn't work in Firefox

### ISSUE THREAD

**Issue by TitanNano** [geek1011/Authenticator#1]
QR code scanning doesn't work in Firefox, works fine in chrome though. 
There are a few errors thrown after granting camera access.

**Comment by geek1011** [comment]
Yep. The QR scanning has been buggy in recent browser versions, but I don't really use this app anymore, so I haven't gotten around to fixing it.
